frontend/src/common/queries/curation.ts

```diff
@@ -0,0 +1,73 @@
+import { useQuery, UseQueryResult } from "react-query";
+import { ENTITIES } from "./entities";
+
+export interface APIKeyResponse {
+  id: string;
+  key?: string;
+}
```

Comment:
@danieljhegeman Here I defined the shape for both ID and Key responses.

In the current swagger draft I think the ID response is a string and not JSON, I thought it made more sense for both responses from the same endpoint to share the same shape